generators/app/templates/_.npmignore

```diff
@@ -0,0 +1 @@
+!dist/
```

Comment:
This is to override `.gitignore`'s exclusion of `dist/` - I think it means everything would be included.